src/main/java/emu/grasscutter/game/mail/BirthdayMailSystem.java

```diff
@@ -0,0 +1,137 @@
+package emu.grasscutter.game.mail;
+
+import emu.grasscutter.Grasscutter;
+import emu.grasscutter.game.player.Player;
+import emu.grasscutter.game.props.ActionReason;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * System for handling birthday gifts for players.
+ * Checks if today is a player's birthday and sends a gift if it is.
+ */
+public class BirthdayMailSystem {
+    // Mail content constants
+    private static final String MAIL_TITLE = "Best Wishes on Your Birthday";
+    private static final String MAIL_CONTENT = "Happy Birthday, Traveler! Please find your gift attached to this message. Thank you for all your support. We wish you a brighter future, no matter what you are going through.";
```

Comment:
This part should be localized using the language system; not all players speak English